pkg/manager/member/pd_failover.go

```diff
@@ -63,20 +63,15 @@ func (f *pdFailover) Failover(tc *v1alpha1.TidbCluster) error {
 			ns, tcName, healthCount, tc.PDStsDesiredReplicas(), tc.Spec.PD.Replicas, len(tc.Status.PD.FailureMembers))
 	}
 
-	failureReplicas := tc.GetPDFailureReplicas()
-	if failureReplicas >= *tc.Spec.PD.MaxFailoverCount {
-		klog.Errorf("PD failover replicas (%d) reaches the limit (%d), skip failover", failureReplicas, *tc.Spec.PD.MaxFailoverCount)
+	deletedFailureReplicas := tc.GetPDFailureReplicas()
```

Comment:
How about giving `GetPDFailureReplicas` a better name?

Reply:
We can use CountPDFailureReplicasTobeDeleted. I will update the changes.

Reply:
fixed